data/mods/PKs_Rebalance/pk_terrain.json

```diff
@@ -0,0 +1,108 @@
+[
+    {
+        "type" : "terrain",
+        "id" : "t_guardrail_bg_dp",
+        "edit-mode": "modify",
+        "name": "guard rail",
+        "symbol": "#",
+        "color": "ltgray",
+        "move_cost": 3,
+        "flags": ["TRANSPARENT", "NOITEM", "REDUCE_SCENT", "MOUNTABLE", "SHORT", "THINOBSTACLE", "ROAD"],
```

Comment:
@mugling sure. BTW, previously I did similar flags check using bash script with ```JSON_FLAGS.md``` as a dictionary but your implementation should be more robust and reliable, I guess.